lib/lookup.json

```
@@ -169,7 +169,8 @@
   },
   "express-session": {
     "prefix": "v",
-    "maintainers": "dougwilson"
+    "maintainers": "dougwilson",
```

```suggestion
    "maintainers": "ulisesGascon",
```

We can update the maintainer too if we want ([ref](https://github.com/expressjs/express/blob/master/Contributing.md#active-projects-and-captains))